contracts/VaultV3.vy

```diff
@@ -157,17 +153,23 @@ name: public(String[64])
 # ERC20 - symbol of the token
 symbol: public(String[32])
 
+full_profit_unlock_date: public(uint256)
+profit_unlocking_rate: uint256
```

Comment:
would it make sense to make this variable public? 

So there is a way to see how much buffer there is without looking at the Vault shares